php/Dockerfile

```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
 MAINTAINER Damien Carcel <damien.carcel@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV GOSU_VERSION 1.9
+
+# Install user impersonator
+RUN set -x \
+    && apt-get update && apt-get install -y --no-install-recommends ca-certificates wget && rm -rf /var/lib/apt/lists/* \
+    && dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" \
```

Comment:
As you need wget, make it like PHP 7 images: install the various tools (wget included), then add the repo for GOSU, then install gosu, then install PHP (each then being a RUN instruction).